great-idea/css/index.css

```diff
@@ -112,7 +112,7 @@ section.services h1 {
 /* Call To Action Selectors (index.html) */
 
 section.cta {
-	background-image: url(C:/Users/kingmoc/Documents/git/User-Interface/great-idea-website/img/header-img.png);
+	background-image: url("../img/header-img.png");
```

Comment:
this path should probably be "./img/header-img.png" since it is going to be reading from where the index.html file is located